src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -343,7 +375,6 @@
     <PropertyGroup Condition="'$(SkipVersionGeneration)'!='true'">
       <BuildNumberMajor >$(GeneratedBuildNumberMajor)</BuildNumberMajor>
```

Comment:
What's the mechanism that makes it only run when the file doesn't exist?  Actually I don't see anywhere that ShouldCreateVersionFileDuringBuild is set, so maybe it was never even running.